.github/workflows/Basic.yml

```diff
@@ -44,6 +44,19 @@ jobs:
       - name: 🦾 Compile WASM contract
         run: cargo build -p seda-contract --release --lib --target wasm32-unknown-unknown --locked
 
+      - name: ☁️ Install cargo-llvm-cov
+        uses: taiki-e/install-action@cargo-llvm-cov
+
+      - name: 📊 Test Coverage
+        run: cargo cov-ci
+
+      - name: 📤 Upload coverage to Codecov
+        uses: codecov/codecov-action@v5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }} # not required for public repos
```

Comment:
The repo is now public so not sure why it fails?

Reply:
Org repo's don't get to take advantage of that :/

Reply:
Removed that comment so it's not confusing